tests/unit/test_config_factory.py

```
@@ -460,6 +460,105 @@ def test_valid_url_with_path_accepted(self):
         )
         assert config.base_url == "http://localhost:1337/api/v1"
 
+    def test_trailing_api_suffix_stripped(self):
+        """Test that a trailing /api path is stripped from base_url."""
+        config = ConfigFactory.create(
+            base_url="https://cms.example.com/api",
+            api_token="test-token",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "api_token"

(S106)